code/search/SearchIndex.php

```diff
@@ -37,6 +37,27 @@
      */
     private static $hide_ancestor;
 
+    /**
+     * Used to separate class name and relation name in the sources array
+     * this string must not be present in class name
+     * @var string
+     * @config
+     */
+    private static $class_delimiter = '_|_';
+
+    /**
+     * This is used to clean the source name from suffix
+     * suffixes are needed to support multiple relations with the same name on different page types
+     * @param string $source
+     * @return string
+     */
+    private function getSourceName($source)
```

Comment:
We don't make functions private usually, please mark it protected unless there's a really good reason that a subclass shouldn't be able to alter this behaviour